### ISSUE TITLE
[feature] log_dir Nomad Config

### ISSUE THREAD

**Issue by jozef-slezak** [hashicorp/nomad#6174]
I would like to propose a feature request: the possibility to define Nomad Config log_dir (other than allocs dir). This would enable us:
1. easily zip all logs from one OS
2. to have dedicate file system / partition /mount

Related to:
https://github.com/hashicorp/nomad/issues/1469
https://github.com/hashicorp/nomad/blob/master/client/logmon/logmon.go#L25

**Comment by cgbaker** [comment]
Yes, I can definitely see how that would be a helpful feature. One benefit of putting logs into the allocation directly is that we have some ability to account for log space usage when scheduling.

**Comment by cgbaker** [comment]
Some of these use cases will be address by the logging plugin system, which is still being designed.

**Comment by mr-karan** [comment]
Is there any more news on this?
